src/instructions/sgx/mod.rs

```diff
@@ -0,0 +1,11 @@
+pub mod enclu;
+
+/// Error codes shared by ENCLS, ENCLU and ENCLV.
+#[derive(Copy, Clone, Debug)]
+#[repr(u8)]
+pub enum EnclaveError {
+    /// All CPU cores have not exited from the previous grace period.
+    pub const PageNotTracked = 11;
+    /// Attributes of the destination page do not match to the secinfo.
+    pub const PageAttributesMismatch = 19;
+}
```

Comment:
This module is currently not included, you need to add the following line to `src/instructions/mod.rs`:
```rust
pub mod sgx;
```